### PR TITLE
rfa 1b set up

### PR DESCRIPTION
**Pull request by lauherkcambria**
#### What tests were added (if relevant), and do all tests (rspec, karma, and integration CALS tests) pass?
-

#### Which User story does this relate to?
- https://www.pivotaltracker.com/story/show/151078075

#### Brief feature description
- rfa o1b default url of rfa/b01 with default cards and layout

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
-n/a

#### Code coverage percent diff as a result of this PR?
-n/a

#### JS errors diff as a result of this PR?
- n/a

### CODE REVIEW

**Comment by lauherkcambria** [comment]
jenkins, retest this please

**Comment by lauherkcambria** [comment]
sounds good~ 😄

File: app/javascript/rfa_forms/rfa01b_list/outsideCACriminalBackground.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                onChange={(event) => this.props.onFieldChange()} />
+              {crimes.map((crime, index) => {
+                return (
+                  <CriminalFields
+                    onFieldChange={this.props.onFieldChange()} />
+                )
+              })}
+            </div>
+            <div className='text-center'>
+              <button onClick={this.addCard} className='btn btn-default'>Add Another O^_ffense - Outside of California +</button>
```
Comment (Nasir2712) [inline app/javascript/rfa_forms/rfa01b_list/outsideCACriminalBackground.js]: text change on this line

File: app/javascript/rfa_forms/rfa01b_list/californiaCriminalBackground.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                onChange={(event) => this.props.onFieldChange()} />
+              {crimes.map((crime, index) => {
+                return (
+                  <CriminalFields
+                    onFieldChange={this.props.onFieldChange()} />
+                )
+              })}
+            </div>
+            <div className='text-center'>
+              <button onClick={this.addCard} className='btn btn-default'>Add Another Off^_ense - California (only) +</button>
```
Comment (Nasir2712) [inline app/javascript/rfa_forms/rfa01b_list/californiaCriminalBackground.js]: And text change on this line too

**Comment by lauherkcambria** [comment]
jenkins, retest this please.

**Pull request merged**